archetypes/archetype_definition_qby_msp.json

```diff
@@ -1,10 +1,9 @@
 {
     "qby_msp": {
         "policy_assignments": [
-            "QBY-Deploy-VM-Backup",
             "QBY-Deploy-DomainJoin",
-            "Deploy-VM-Monitoring",
-            "QBY-Deploy-LAW-Agent"            
+            "QBY-Deploy-VM-Backup",
```

Comment:
We only monitor and backup VMs, that are managed by us? :O

Reply:
Yes. That is the default. If the customer want's this too we will gladly deploy the policies in their MGs also